### PR TITLE
feat: enhance readme to fix #46

### LINKED ISSUE #46: Improve README.md
We need to add a tutorial on how to use Docker. Plus, I think the project idea can be better explained.

### PR DESCRIPTION
**Pull request by peterjgrainger**
### What does this implement/fix? ###
Change the readme file to be more consistent with other readme's on github.  Added a section on how to set up docker and clarified what the repo is.

### Does this close any currently open issues? ###
closes #46

### Explain your solution ###
Used this: https://gist.github.com/PurpleBooth/109311bb0361f32d87a2 as a starting point and added sections.  Explained a bit more about the project and put up a screenshot of the site 

I'm open to any changes.  Let me know.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,44 +29,102 @@
...
+
+## Tests
+
+```bash
+rails test
+```
+
+## Contribute
+
+Fork and make a pull request
```
Comment (MatheusRich) [inline README.md]: Add a link to the CONTRIBUTING.md file (it's on `.github/`)

File: README.md [hunk 2]
```diff
@@ -20,44 +29,102 @@
...
+
+## Credits
+
+The idea is thanks to [dontpad](http://dontpad.com)
```
Comment (MatheusRich) [inline README.md]: Dontfile was inspired by [dontpad](http://dontpad.com) philosophy.

File: README.md [hunk 3]
```diff
@@ -20,44 +29,102 @@
...
+
+## License
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
```
Comment (MatheusRich) [inline README.md]: Change to:
Dontfile is under the [MIT License](LICENSE)! You can download, change, share the source code!

**Comment by peterjgrainger** [comment]
Applied all the comments.  Please have a look again

**Comment by MatheusRich** [comment]
@peterjgrainger Thanks for the contribution :heart_eyes:! Merging it!

**Pull request merged**